app/clients/cbc_proxy.py

```diff
@@ -118,6 +122,10 @@ def _invoke_lambda_with_failover(self, payload):
     def _invoke_lambda(self, lambda_name, payload):
         payload_bytes = bytes(json.dumps(payload), encoding='utf8')
         try:
+            current_app.logger.info(
+                f"Calling lambda {lambda_name} for link test with payload {payload}"
```

Comment:
It looks to me like this log line wouldn't just get used for link tests, but for all broadcasts (since `_invoke_lambda` is used for real broadcasts too) - is that correct?

If so having "link test" in the message is a bit confusing, plus the payload could be very large (though we do log the whole payload if there's an error, so that's not necessarily an issue)

Reply:
Good spot thank you. This function is also called when sending regular messages. I've amended the log line 👍